summary_workflows/quantification/quantification_dockers/q2_validation/validation.py

```diff
@@ -67,7 +76,7 @@ def  validate_input_data(input_participant,  public_ref_dir, community, challeng
                 methods_check = list(public_ref_data.iloc[:, 0].values)
```

Comment:
The file public_ref_data is never used for validation. Instead the validation checks (apparently checking whether the input file format is valid bed) are coded here in this script.    
I find this slightly confusing, can it be cleaned up? E.g. remove all parts about the public_ref, especially the for loop we are in here? The checks performed here have nothing to do with the files present in public_ref_dir. But then again, the OEB schema might be unhappy if we don't use a public_ref file, isn't it? Any thoughts @AsierGonzalez ?

Reply:
As we discussed on Slack, we can safely remove all references to `public_ref_data`, related variables and the the file [public_ref.txt](https://github.com/iRNA-COSI/APAeval/blob/add_working_Q2_summaryworkflow/summary_workflows/quantification/APAeval_Q2_test_data/public_ref/public_ref.txt) itself.